src/app/core/services/issue.service.ts

```diff
@@ -483,4 +483,26 @@ export class IssueService {
   getIssueTeamFilter(): string {
     return this.issueTeamFilter;
   }
+
+  /**
+   * Returns a customised error message for operations with 1 Issue
+   */
+  private getIssueCustomErrorMessage(error) {
+    const issueErrors = error.response.data.errors;
```

Comment:
error.response is returning ```undefined ``` which throws ```TypeError: Cannot read properties of undefined (reading 'data'). ``` This error is unhandled (because it is thrown within our customised error handler) thus I am getting the error messages as shown in my screenshots. 

Please look into this!